pkgs/applications/science/biology/jellyfish/default.nix

```
@@ -1,14 +1,22 @@
-{ stdenv, fetchurl, lib }:
+{ stdenv, fetchFromGitHub, lib, autoconf, automake, libtool, gettext, pkg-config }:
 
 stdenv.mkDerivation rec {
     pname = "jellyfish";
     version = "2.3.0";
 
-    src = fetchurl {
-        url = "https://github.com/gmarcais/Jellyfish/releases/download/v${version}/${pname}-${version}.tar.gz";
-        sha256 = "e195b7cf7ba42a90e5e112c0ed27894cd7ac864476dc5fb45ab169f5b930ea5a";
+    src = fetchFromGitHub {
+        owner = "gmarcais";
+        repo = pname;
+        rev = "v${version}";
+        sha256 = "1d9f0ni2ll1basc5p0shwb4gq1iz1ahnzv2imfgb93bihyvcmq66";
     };
 
+    buildInputs = [ autoconf automake libtool gettext pkg-config ];
+
+    preConfigure = ''
+        autoreconf -i
+    '';
```

```suggestion
    nativeBuildInputs = [ autoreconfHook gettext pkg-config ];
```
is gettext used as a binary or cli tool? If it is the first it needs to go to buildInputs.